packages/dcos-signal/buildinfo.json

```diff
@@ -1,5 +1,4 @@
 {
-  "docker": "golang:1.6",
```

Comment:
the repo / it's tests don't run docker 1.7 currently, need to change that before we swap this out, make sure the in-repo tests pass under go 1.7